.editorconfig

```diff
@@ -15,3 +15,6 @@ indent_size = 2
 
 [Makefile]
 indent_style = tab
+
+[CHANGELOG.md]
+trim_trailing_whitespace = false
```

Comment:
you might want to add this in the master branch - and maybe in all repos

Reply:
definitely.  will add a jira ticket for this.

Reply:
https://jira.corp.stripe.com/browse/DEVSDK-2229